File: _old/5_Vowal_String_Count/Golang/Main.go
package main

import (
	"fmt"
	"strings"
)

func vowalString(words []string, left, right int) int {
	var vowals = "aeiou"
	var count = 0

	for i := left; i <= right; i++ {
		spString := strings.Split(words[i], "")

		if strings.Contains(vowals, spString[0]) && strings.Contains(vowals, spString[len(spString)-1]) {
			count++
		}
	}

	return count
}

func main() {

	x := vowalString([]string{"hey", "aeo", "mu", "ooo", "artro"}, 1, 4)
	fmt.Println(x)
}

File: _old/5_Vowal_String_Count/JavaScript/Main.js
var vowalString = function (words, left, right) {
  let vowals = "aeiou"
  let count = 0

  for (let i = left; i <= right; i++ ) {
    let spString = words[i].split('')

    if (vowals.includes(spString[0]) && vowals.includes(spString[spString.length - 1])) {
      count++
    }
  }
  
  return count
};

console.log(vowalString(["hey", "aeo", "mu", "ooo", "artro"], 1, 4));